client/components/upgrades/gsuite/README.md

```diff
@@ -6,19 +6,12 @@ GSuiteUpgrade is a React component used to add G Suite email addresses to domain
 
 ```jsx
 import React from 'react';
-import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
 import GSuiteUpgrade from 'calypso/components/upgrades/gsuite';
 import productsListFactory from 'calypso/lib/products-list';
 
 const productsList = productsListFactory();
 
-class MyComponent extends React.Component {
-	render() {
-		return (
-			<CalypsoShoppingCartProvider>
-				<GSuiteUpgrade domain={ domain } />;
```

Comment:
I wonder how the linter was not failing on this one with that semicolon at the end of the line 🤔

Reply:
This is JSX: the semicolon was not a JS token, but a string that was rendered into the DOM markup.